x/evm/module.go

```diff
@@ -188,5 +188,7 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 
 // WeightedOperations returns the all the evm module operations with their respective weights.
```

Comment:
we need to add the functions above too

- `RandomizedParams`
- `RegisterStoreDecoder`
- `ProposalContents`

Reply:
`RandomizedParams` is still missing